utils/error-handler.tsx

```diff
@@ -0,0 +1,53 @@
+import axios from "axios";
+
+import { CodedError } from "@utils/coded-error";
+import { ErrorProps } from "@components/figure/ErrorMessage";
+
+/**
+ * Error handler for the Verify page (pages/verify.tsx)
+ * @param e
+ */
+export const verifyErrorHandler = (e: unknown): ErrorProps => {
+  if (e instanceof CodedError) {
+    return {
+      type: "ERROR",
+      message: (
+        <div className="text-center">
+          {e.type}: {e.message}
+          {e.details && <br />}
+          {e.details && <span className="break-words">{e.details}</span>}
+        </div>
+      ),
+    };
+  } else if (axios.isAxiosError(e)) {
+    const isSpmTransientStorage = (url = "") => url?.startsWith("https://api-spm-vac.storage.aws.notarise.gov.sg");
```

Comment:
Need to update this liner to check for all possible SPM transient storage URLs (both production and staging).